thesis-api-v2/src/main/java/pl/edu/agh/quizzesthesis/data/repository/TermRepository.java

```diff
@@ -28,4 +30,7 @@ public interface TermRepository extends PagingAndSortingRepository<Term, Integer
 
     @Transactional
     void deleteAllByCategory(Category category);
+
+    @Query(nativeQuery = true, value = "SELECT * FROM term WHERE category_id = ?1 ORDER BY ABS(difficulty - ?2) ASC LIMIT 50")
```

Comment:
Ideally you would want to adjust `LIMIT` based on number of terms in category. For me, it can stay as it is since it is clean and simple. We can adjust the number in the future if we have to, but for now it can stay that way.

Reply:
I would leave it as it is and adjust this in another task if we have time.

Reply:
I am ok with that.